pkg/controller/yurthub/crsapprover.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yurthub
```

Comment:
package name `yurthub` is component name and maybe makes user  confused, so use `the function of this package` will be more understandable. so how about rename the package name to `certificates`?

Reply:
and file name has a typo error. `crsapprover.go` --> `csrapprover.go`